packages/devtools/lib/src/timeline/timeline.css

```diff
@@ -68,6 +67,7 @@
     position: absolute;
     white-space: nowrap;
     height: 24px;
+    z-index: 1;
```

Comment:
I'm fine with this either way...but chrome devtools does show the lines through the events.
![screen shot 2019-03-06 at 1 36 54 pm](https://user-images.githubusercontent.com/43759233/53915595-19a14100-4015-11e9-9709-cc95521f48b4.png)

Reply:
Gotcha; I like the look of more solid (opaque) timeline events, but this is your UI, so I'm happy either way -

Reply:
I do think the lines showing through helps you see that something is happening when you are really zoomed in and still zooming. The moving lines are your indicator that something is happening. 

*If you're looking at the flame chart on a smaller screen and the event tiles take up the entire height of the flame chart container, and you are in the super zoomed in case, removing the lines brings us back to the original problem of not being able to tell when you're zooming in/out.

While looking at this, I also noticed that the top div with the timestamps isn't sticky, and it should be. Having this be sticky would help with the above described case, but still wouldn't be ideal.